### DISCUSSION TITLE
Tuya RTSP or HLS streaming

### DISCUSSION THREAD

**Discussion by seekoner** [Sunoo/homebridge-camera-ffmpeg#1259] [Q&A]
Hello, I managed to get my tuya ipcamera streaming urls in the tuya dev lot but can't make it work with FFMPEG I guess it's because it's a cloud based encryption thing.
But I wonder if any of you guys managed to make it work on this plugin?

Thanks ![image](https://user-images.githubusercontent.com/98288086/151876431-ed019f39-d201-4614-8b4b-60bf721806fe.png)

**Comment by Sunoo** [comment]
I know @donavanbecker has been looking into the Tuya stuff, maybe he can speak more definitively. It was my understanding that the URLs change like every 10 minutes or so though.

**Reply by donavanbecker** [reply]
Yes this is a WIP.

**Reply by nmeliasp** [reply]
@donavanbecker has there been any progress? I am in the same boat here where my rstp URL doesnt work with this plugin

**Reply by donavanbecker** [reply]
use this plugin: https://github.com/0x5e/homebridge-tuya-platform#readme

**Comment by donavanbecker** [comment]
https://github.com/0x5e/homebridge-tuya-platform#readme

**Reply by nmeliasp** [reply]
bummer this isnt working, getting a few error messages once i provided the authentication/basic configs for the plug-in

**Reply by donavanbecker** [reply]
> bummer this isnt working, getting a few error messages once i provided the authentication/basic configs for the plug-in

I would report an issue to the plugin then.
